pkg/test/utils.go

```diff
@@ -169,50 +169,18 @@ func CompareGoldenObject(t *testing.T, p string, got []byte) {
 		t.Errorf("Failed parsing file: %s", err)
 	}
 
-	diff := cmp.Diff(postWriteNormalization(wantMap), gotMap)
+	diff := cmp.Diff(wantMap, gotMap)
 	if diff == "" {
 		return
 	}
 
+	t.Errorf("unexpected diff in %s: %s", p, diff)
+
 	if writeGoldenOutput {
 		// Write the output to the golden file
 		if err := os.WriteFile(p, []byte(got), 0644); err != nil {
 			t.Fatalf("failed to write golden output %s: %v", p, err)
 		}
-		t.Errorf("wrote updated golden output to %s", p)
-	} else {
-		t.Errorf("unexpected diff in %s: %s", p, diff)
-	}
-}
-
-func postWriteNormalization(wantMap map[string]interface{}) map[string]interface{} {
```

Comment:
+1 to not override the test expectations.
What is the current LLM behavior if user specify those two annotations? Is it a no-op or we have some handling in the direct controller(I don't recall I see any, just to double check in case I miss anything)?

I recall in SIS, we decide to add some warnings if user specify the unexpected annotations.

Reply:
It's a no-op, I believe, but @acpana can confirm